spec.html

```diff
@@ -15478,16 +15404,72 @@ <h1>Runtime Semantics: Evaluation</h1>
         1. Let _lval_ be ? GetValue(_lref_).
         1. Let _rref_ be the result of evaluating |AssignmentExpression|.
         1. Let _rval_ be ? GetValue(_rref_).
-        1. Let _op_ be the `@` where |AssignmentOperator| is `@=`.
-        1. Let _r_ be the result of applying _op_ to _lval_ and _rval_ as if evaluating the expression _lval_ _op_ _rval_.
+        1. Let _assignmentOpText_ be the source text matched by |AssignmentOperator|.
+        1. Let _opText_ be the result of removing the final `=` from _assignmentOpText_.
```

Comment:
This is fine -- certainly better than what we have now -- but I was envisioning we would just have a table from assignment operator to infix operator. @tc39/ecma262-editors which do you prefer?

Reply:
I think I prefer a lookup table to what's effectively string manipulation.

Reply:
Okay, I've replaced the text-manipulation with a lookup table.